src/extremeweatherbench/utils.py

```diff
@@ -481,6 +481,7 @@ def align_point_obs_from_gridded(
                 valid_time_index, :
             ]
         else:
+            logger.debug("No valid time found in point obs for %s", valid_time)
```

Comment:
Probably want to log the statement `valid_time.strftime(...)` with some format statement that will make this easier to read.